subcommands/devices/config_rotate_certs.go

```diff
@@ -16,7 +16,7 @@ func init() {
 		Short: "Rotate a device's x509 keypair used to connect to the device gateway",
 		Args:  cobra.ExactArgs(1),
 		Run:   doConfigRotate,
-		Long: `This command will send a fioconfig change to a device to instruct it to perform
+		Long: `This command will send a fioconfig change to a device instructing it to perform
```

Comment:
Do we need a comma between `device, instructing`?
This sentence sounds quite overloaded without a comma.
(I would not comment, but I noticed a comma added in a similar sentence in `config_set.go` line 23).